config.m4

```diff
@@ -273,6 +308,9 @@ if test "$PHP_MONGODB" != "no"; then
     _include([scripts/autotools/libmongoc/Versions.m4])
     _include([scripts/autotools/libmongoc/WeakSymbols.m4])
 
+    _include([scripts/autotools/libmongocrypt/CheckSSL.m4])
```

Comment:
I'd suggest a comment above this to clarify that `PHP_CLIENT_SIDE_ENCRYPTION` will be set to "yes" or "no" after this include (we can ignore the case where it fails). That will preemptively address any question of why we don't bother to consider "auto" after this point.